packages/core/scripts/build/copyFiles.ts

```diff
@@ -31,6 +31,11 @@ const tasks: CopyTask[] = [
     { target: path.join('src', 'testFixtures') },
     { target: 'src/auth/sso/vue' },
 
+    // Vue.js for webviews
+    {
+        target: path.join('../../node_modules', 'vue', 'dist', 'vue.global.prod.js'),
```

Comment:
please test both Q and toolkit

Reply:
verified both Q and toolkit can login of the generated vsix.  following is the toolkit login, Q login pasted in the pr description
<img width="506" alt="image" src="https://github.com/user-attachments/assets/d180ef5e-d326-47e3-bd85-4c9d596620b4" />